pkg/controllers/vdb/obj_reconciler_test.go

```diff
@@ -46,7 +46,7 @@ var _ = Describe("obj_reconcile", func() {
 
 	runReconciler := func(vdb *vapi.VerticaDB, expResult ctrl.Result, mode ObjReconcileModeType) {
 		// Create any dependent objects for the CRD.
-		pfacts := MakePodFacts(vdbRec, &cmds.FakePodRunner{}, logger)
+		pfacts := MakePodFacts(vdbRec, &cmds.FakePodRunner{}, logger, "")
```

Comment:
```suggestion
		pfacts := MakePodFacts(vdbRec, &cmds.FakePodRunner{}, logger, vapi.MainCluster)
```
minor: use v1.MainCluster const in place of empty sandbox name

Reply:
I think this new empty string is for the dbadmin password. It might be clear if we still used a const for it. i.e. TestEmptyPassword